spec/kitchen/provisioner_spec.rb

```diff
@@ -47,6 +47,21 @@ class Coolbeans < Kitchen::Provisioner::Base
       provisioner.must_be_kind_of Kitchen::Provisioner::Coolbeans
     end
 
+    it "returns a provisioner object of the chef infra for chef zero" do
+      provisioner = Kitchen::Provisioner.for_plugin("chef_zero", {})
+      provisioner.must_be_kind_of Kitchen::Provisioner::ChefInfra
+    end
+
+    it "returns a provisioner object of the chef infra for chef_infra" do
+      provisioner = Kitchen::Provisioner.for_plugin("chef_zero", {})
```

Comment:
Description and plugin don't match - it says 'chef_infra' is given but we're giving 'chef_zero'.